hal/inc/wlan_hal.h

```diff
@@ -413,8 +428,9 @@ int wlan_restart(void* reserved);
 int wlan_set_hostname(const char* hostname, void* reserved);
 int wlan_get_hostname(char* buf, size_t buf_size, void* reserved);
 
-int wlan_set_country_code(const char * country_code);
-int wlan_get_country_code(char * country_code);
+wlan_country_code_t wlan_get_country_code(void* reserved);
+int wlan_set_country_code(wlan_country_code_t country, const void* reserved);
```

Comment:
Not sure why we've added `const` :) Reserved arguments are all just `void*`, see above.